src/parser/generated_parser.ts

```diff
@@ -6004,43 +6005,41 @@ export class GeneratedParser extends Parser {
 
         return null;
     }
-
-    get keywords(): Map<string, KeywordToken> {
-        return new Map([
-            ["return", new KeywordToken("return", 500)],
-            ["raise", new KeywordToken("raise", 501)],
-            ["pass", new KeywordToken("pass", 502)],
-            ["del", new KeywordToken("del", 503)],
-            ["yield", new KeywordToken("yield", 504)],
-            ["assert", new KeywordToken("assert", 505)],
-            ["break", new KeywordToken("break", 506)],
-            ["continue", new KeywordToken("continue", 507)],
-            ["global", new KeywordToken("global", 508)],
-            ["nonlocal", new KeywordToken("nonlocal", 509)],
-            ["if", new KeywordToken("if", 510)],
-            ["try", new KeywordToken("try", 511)],
-            ["while", new KeywordToken("while", 512)],
-            ["import", new KeywordToken("import", 513)],
-            ["from", new KeywordToken("from", 514)],
-            ["elif", new KeywordToken("elif", 515)],
-            ["else", new KeywordToken("else", 516)],
-            ["for", new KeywordToken("for", 517)],
-            ["in", new KeywordToken("in", 518)],
-            ["with", new KeywordToken("with", 519)],
-            ["as", new KeywordToken("as", 520)],
-            ["except", new KeywordToken("except", 521)],
-            ["finally", new KeywordToken("finally", 522)],
-            ["def", new KeywordToken("def", 523)],
-            ["class", new KeywordToken("class", 524)],
-            ["lambda", new KeywordToken("lambda", 525)],
-            ["not", new KeywordToken("not", 526)],
-            ["is", new KeywordToken("is", 527)],
-            ["True", new KeywordToken("True", 528)],
-            ["False", new KeywordToken("False", 529)],
-            ["None", new KeywordToken("None", 530)],
-            ["__peg_parser__", new KeywordToken("__peg_parser__", 531)],
-            ["or", new KeywordToken("or", 532)],
-            ["and", new KeywordToken("and", 533)],
-        ]);
-    }
 }
+
+GeneratedParser.prototype.keywords = new Map<string, KeywordToken>([
+    ["return", new KeywordToken("return", 500)],
```

Comment:
Feel free to change this.
The `get keyword() {` wasn't quite right because it would create a new Map for every access to `p.keywords`

Reply:
ah yeah it was just a way to get it to work